lib/statsd/instrument/rubocop/metric_value_keyword_argument.rb

```diff
@@ -0,0 +1,42 @@
+# frozen-string-literal: true
+
+module RuboCop
+  module Cop
+    module StatsD
+      # This Rubocop will check for providing the value for a metric using a keyword argument, which is
+      # deprecated. Use the following Rubocop invocation to check your project's codebase:
+      #
+      #    rubocop --require \
+      #      `bundle show statsd-instrument`/lib/statsd/instrument/rubocop/metric_value_keyword_argument.rb \
+      #      --only StatsD/MetricValueKeywordArgument
+      #
+      # This cop will not autocorrect offenses. Most of the time, these are easy to fix by providing the
+      # value as the second argument, rather than a keyword argument.
+      #
+      # `StatsD.increment('foo', value: 3)` => `StatsD.increment('foo', 3)`
+      #
+      class MetricValueKeywordArgument < Cop
+        MSG = 'Do not use the value keyword argument, but use a positional argument'
+
+        STATSD_METRIC_METHODS = %i{increment gauge measure set histogram distribution key_value}
+
+        def on_send(node)
+          if node.receiver&.type == :const && node.receiver&.const_name == "StatsD"
+            if STATSD_METRIC_METHODS.include?(node.method_name)
+              arguments = node.arguments
+              arguments.pop if node.arguments.last.type == :block_pass
```

Comment:
Is it safe to mutate `node.arguments`?  Couldn't that affect other cops?

Reply:
I'll add a dup just in case.

Reply:
Rather than dupping I changed it to simply extract the last argument into a variable.